__tests__/components/templates/SinopiaResourceTemplates.test.js

```diff
@@ -6,7 +6,7 @@ import { mount, shallow } from 'enzyme'
 import SinopiaResourceTemplates from 'components/templates/SinopiaResourceTemplates'
 import { getEntityTagFromGroupContainer, getResourceTemplate, listResourcesInGroupContainer } from 'sinopiaServer'
 
-jest.mock('../../../src/sinopiaServer')
+jest.mock('sinopiaServer')
```

Comment:
I like to do `import * as server from 'sinopiaServer'`.  Then you can do `server.getResourceTemplate = jest.fn()`

Reply:
i like justn's suggestion - it reads better.

Reply:
noted this in a comment on #838.  will merge this since @jcoyne approved, and will try that suggestion in follow on work.